### PR TITLE
micronaut 2.4

### PR DESCRIPTION
**Pull request by pgressa**
Note: This PR is not meant to be merged. While working `coherence-bom` I  have found out that the few coherence tests are failing when microanut 2.4. So instead to eliminate the changes made to gradle due to `coherecen-bom` I created this PR.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,12 +15,20 @@ subprojects { Project subproject ->
...
+        jcenter()
+        mavenCentral()
         maven {
             url "https://oss.sonatype.org/content/repositories/snapshots"
             mavenContent {
                 snapshotsOnly()
             }
         }
+        maven {
+            url "https://s01.oss.sonatype.org/content/repositories/snapshots"
```
Comment (pgressa) [inline build.gradle:27]: micronaut 2.4.0-SNAPSHOT is here (don't know why)

**Comment by graemerocher** [comment]
@thegridman could you sign the CLA?

**Comment by pgressa** [comment]
@rlubke could you sign the CLA so we can merge this and move forward, thanks!

**Comment by graemerocher** [comment]
@rlubke could you sign the CLA?

**Comment by rlubke** [comment]
CLA has been signed.

**Comment by graemerocher** [comment]
@pgressa before merging this we need to revert the micronaut version to 2.3.4

**Pull request closed without merging**